encoder.go

```diff
@@ -1,10 +1,23 @@
 package zerolog
 
 import (
+	"encoding/json"
 	"net"
 	"time"
 )
 
+var (
+	// JSONMarshaler is used to marshal interface
+	// to JSON encoded byte slice when an interface
+	// is passed for logging.
+	// Default: "encoding/json.Marshal"
+	JSONMarshaler InterfaceMarshaler = json.Marshal
```

Comment:
I would rather have this variable named `InterfaceMarshalFunc` and remove `InterfaceMarshaler` type that is not very useful. Please update the comment so it does not wrongly implies this is an interface.

Also, please move this to the `globals.go` file.

Reply:
I agree. Will push another fix till all coversation resolved.